holoviews/core/util.py

```diff
@@ -1327,7 +1328,9 @@ def drop_streams(streams, kdims, keys):
     stream_params = stream_parameters(streams)
     inds, dims = zip(*[(ind, kdim) for ind, kdim in enumerate(kdims)
                        if kdim not in stream_params])
-    return dims, [tuple(wrap_tuple(key)[ind] for ind in inds) for key in keys]
+    get = itemgetter(*inds)
```

Comment:
Don't often see a use for ``itemgetter`` and I'm trying to think whether you really need it here. I don't immediately have a better suggestion so it is probably ok...

Reply:
itemgetter is about 100x faster than the previous tuple comprehension.

Reply:
That seems to be a good reason. Could you put in a comment to that effect?

Reply:
Sure, that said itemgetter is used frequently throughout utils, so I'm not sure this one deserves a special note.